### PR TITLE
IA/IL/SD Updates from Highway63

### PR DESCRIPTION
**Pull request by Markkos1992**
Updates Entries included for US 40 (IL), SD 46, and SD 50 due to broken list files.

Note that I added asterisks to the OldI-35_S and OldI-35N points near the 94 point on I-35 in IA.

From Highway63:
Covering the most important stuff - reroutes and new exits, plus a handful of stuff in Illinois.

Because I need to add the Forevergreen Road exit on I-380 in Iowa, this will double as my step into numbering exits along non-interstates. US 218 and IA 27 are excellent candidates because they are mostly/entirely four-lanes, if not controlled access. (IA 58 tags along because IA 27/58 is getting new signs in Cedar Falls, including exit numbers.) “Re-entered route with exit numbers” will be a common line when I get more routes done. I wish there could be some extra “Anyone with points on this highway please check their list” note. All the old names are demoted, so still available.

Also note: This breaks the IL140 point on US 40, which from the forum I think is a label for your list.

Also also note: WI 100 sharp angle error is a false positive. (This is in your major issues list.)

Iowa I-35: Added I-80 exit 127A (Meredith Dr).
Iowa I-80: Added exit 127A (Meredith Dr).
Iowa I-380: Added exit 2 (Forevergreen Rd).
Iowa US 6: Reopened I-29 exits 49 and 50 (US 6 on local lanes).
Iowa US 218: Re-entered route with exit numbers. Added I-380 exit 2 (Forevergreen Rd). Renamed I-380_End->MitAve. Closed *OldUS218_Jan.
Iowa IA 27: Re-entered route with exit numbers. Added I-380 exit 2 (Forevergreen Rd).
Iowa IA 58: Re-entered route with exit numbers from US 20 and IA 27.

Illinois I-55: Added shaping point between 4 and 6.
Illinois I-70: Added shaping points between 4 and 6, 61 and 63, 154 and IL/IN line.  (Covers https://forum.travelmapping.net/index.php?topic=3522.msg23448#msg23448)
Illinois I-90: Added exit 36 (IL 23).  (Covers https://forum.travelmapping.net/index.php?topic=4332.0)
Illinois US 34: Added point CanTr.
Illinois US 40: Relocated IL4 point to connector road. Corrected US45_S from 4th Street to Banker Street. Renamed IL140_W->IL140. Previous IL140 point deleted. Added shaping point between exits 4 and 6.  (Covers https://forum.travelmapping.net/index.php?topic=3522.msg23448#msg23448)
Illinois IL 23: Added point I-90.  (Covers https://forum.travelmapping.net/index.php?topic=4332.0)
Illinois IL 127: Added BriSt_N. Corrected SorRd->SorAve. (Covers https://forum.travelmapping.net/index.php?topic=3522.msg23448#msg23448)

South Dakota SD 50: Rerouted from diagonal west of Wagner between Walnut Avenue and 392nd Avenue onto SD 46 and 392nd Avenue. Old point renamed WalAve. Added and closed *OldSD50. Corrected 268thSt-W->268thSt_W.
South Dakota SD 46: Relocated SD50_W from Walnut Avenue to 392nd Avenue west of Wagner. Old point renamed WalAve. Added closed point *SD50.  (Covers https://forum.travelmapping.net/index.php?topic=4327.0)

### CODE REVIEW

File: hwy_data/IA/usaia/ia.ia027.wpt [hunk 1]
```diff
@@ -23,99 +23,103 @@ CRW60 http://www.openstreetmap.org/?lat=41.162881&lon=-91.541862
...
+229(20) +AnsAve http://www.openstreetmap.org/?lat=42.453608&lon=-92.376308
+227(20) +US63 http://www.openstreetmap.org/?lat=42.458579&lon=-92.405105
+225(20) +US20_W http://www.openstreetmap.org/?lat=42.464658&lon=-92.444286
 RidAve http://www.openstreetmap.org/?lat=42.469501&lon=-92.445831
-VikRd http://www.openstreetmap.org/?lat=42.484061&lon=-92.445703
+181 +VikRd http://www.openstreetmap.org/?lat=42.484061&lon=-92.445703
 GreRd http://www.openstreetmap.org/?lat=42.498744&lon=-92.452826
-UnivAve http://www.openstreetmap.org/?lat=42.513108&lon=-92.443342
-18thSt http://www.openstreetmap.org/?lat=42.523262&lon=-92.441154
-US218/57 http://www.openstreetmap.org/?lat=42.537493&lon=-92.425919
```
Comment (rickmastfan67) [inline hwy_data/IA/usaia/ia.ia027.wpt:81]: Issue here.  Highway63 when changing this to '187(218)', changed the hidden label to '+IA57/58'.  Sadly, 'US218/57' is currently in use and needs to be preserved.

File: updates.csv [hunk 2]
```diff
@@ -3841,6 +3841,7 @@ date;region;route;root;description
 2017-04-02;(USA) Idaho;US 95;id.us095;Moved east onto new alignment in Athol, between waypoints RemRd and *OldUS95_N
 2017-04-02;(USA) Idaho;US 95;id.us095;Relocated east from US 2 (5th Avenue), Cedar Street, 1st St, and Superior St onto new Sandpoint bypass between waypoints SupSt and US2/200
 2015-08-03;(USA) Idaho;ID 16;id.id016;Route extended at south end from ID44 to US20/26
+2021-05-19;(USA) Illinois;US 40;il.us040;IL140 point relocated to former location of IL140_W.
```
Comment (rickmastfan67) [inline updates.csv:3844]: This entry should probably mention that the 'Old IL140' point is now 'GalSt_W'.

Maybe change it to the following:
"IL140 point (now GalSt_W) relocated to former location of IL140_W."

**Comment by rickmastfan67** [comment]
@jteresco Two small issues I noticed in this listed above.

**Comment by jteresco** [comment]
I'll hold off merging this until I hear it's good to go.

**Comment by Markkos1992** [comment]
I made both corrections.  @jteresco, could you email Highway63 the updated IL 27 file with this change made?

**Comment by jteresco** [comment]
Is IL 27 the only file unchanged from the emailed zip file?

**Comment by jteresco** [comment]
And we're good now to bring this in tonight?

**Comment by Markkos1992** [comment]
Both I-35 and IL 27 were changed from the emailed zip file.  

Mark Moore

Civil and Environmental Engineering
Virginia Tech Class of 2014
***@***.***
(804)691-4381

**Comment by Markkos1992** [comment]
The files should be good to go.  I am a bit unsure on how to complete the review though.

**Pull request merged**